group_vars/forkhead_cluster/ip_addresses.yml

```diff
@@ -0,0 +1,7 @@
+---
+ip_addresses:
+  betabarrel:
```

Comment:
What is this betabarrel address doing in the list of IPs for Fork Head?

Reply:
Fixed. Thanks.